crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/prosecution_concluded/service/ProsecutionConcludedService.java

```diff
@@ -67,12 +69,15 @@ public void executeCCOutCome(ProsecutionConcluded prosecutionConcluded, WQHearin
     }
 
     private void processOutcome(ProsecutionConcluded prosecutionConcluded, WQHearingDTO wqHearingDTO, List<OffenceSummary> trialOffences) {
-        prosecutionConcludedValidator.validateOuCode(wqHearingDTO.getOuCourtLocation());
+        String crownCourtCode = prosecutionConcludedValidator.validateOuCode(wqHearingDTO.getOuCourtLocation());
```

Comment:
Done, now split this out so that the validator only checks the code exists (by calling a new method `isValidCode` on the CrownCourtCodeHelper) and then later on in the ProsecutionConcludedService getting the code by calling the existing `getCode` method (again on the CrownCourtCodeHelper).